### PR TITLE
Remove 9-patch files and enable ripple effect

### PR DESCRIPTION
**Pull request by LeSalamandre**
- Removes all 9-patch files, replacing them with a single android layout configuration
- Adds a foreground ripple effect to compatible android versions

### CODE REVIEW

File: java/app/src/main/res/drawable/add_to_google_wallet_button_background.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <size android:minWidth="300px" android:minHeight="48px"/>
```
Comment (JlUgia) [inline java/app/src/main/res/drawable/add_to_google_wallet_button_background.xml:4]: Make sure to [always use `dp` on Android](https://developer.android.com/training/multiscreen/screendensities), which are screen density independent units. Using px will naturally make elements be drawn in smaller sizes on screens with a larger pixel density.

**Comment by JlUgia** [comment]
Once we approve the changes, they'll also need to be applied to the Kotlin repo.

File: java/app/src/main/res/layout/add_to_google_wallet_button.xml [hunk 2]
```diff
@@ -5,12 +5,13 @@
     android:layout_width="match_parent"
     android:minWidth="267dp"
     android:layout_height="48dp"
-    android:background="@drawable/add_to_google_wallet_button_background_shape"
+    android:background="@drawable/add_to_google_wallet_button_background"
+    android:foreground="@drawable/ripple_effect"
```
Comment (JlUgia) [inline java/app/src/main/res/layout/add_to_google_wallet_button.xml:9]: Ah, now I know what you mean with the ripple not clipping on the edges. This is easy to achieve adding the ripple to the background.
Also, the foreground property has its own intricacies that make it misbehave on v < 23 (see [official bug](https://b.corp.google.com/issues/37065042?pli=1), [forum discussions](https://stackoverflow.com/questions/52329249/why-doesnt-androidforeground-attribute-work)).

In any case, I wasn't able to see the ripple with the current implementation. Have you considered keeping it in the bg like we do for the `Add to Wallet` button?

File: java/app/src/main/res/drawable/add_to_google_wallet_button_background.xml (hunk 1 above)
Comment (LeSalamandre) [reply java/app/src/main/res/drawable/add_to_google_wallet_button_background.xml]: Updated to use DP.

File: java/app/src/main/res/layout/add_to_google_wallet_button.xml (hunk 2 above)
Comment (LeSalamandre) [reply java/app/src/main/res/layout/add_to_google_wallet_button.xml]: Strange that you couldn't see it, but I've wholly replaced the mechanism to mimic the Wallet button's and re-pushed; it should work now the same way as the wallet button.

**Comment by JlUgia** [comment]
I cleaned up superfluous files, merged conflicts and replicated the changes in the kotlin repo.
@LeSalamandre feel free to give this one last look and merge.

**Pull request merged**